src/LogIn.js

```diff
@@ -0,0 +1,105 @@
+import axios from 'axios';
+import React, { useState, useCallback, useContext } from 'react';
+import { useHistory } from 'react-router';
+import { useCookies } from 'react-cookie';
+import {
+  AppContext,
+  LOG_IN_FAILURE,
+  LOG_IN_REQUEST,
+  LOG_IN_SUCCESS,
+} from './App.js';
+import styled from 'styled-components';
+
+const LogIn = () => {
+  const { dispatch } = useContext(AppContext);
+  const history = useHistory();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [cookies, setCookie, removeCookie] = useCookies(['Token']);
+
+  const handleToSignUp = () => {
+    history.push('/signup');
+  };
+  const handleEmailChange = useCallback((e) => {
+    setEmail(e.target.value);
+  }, []);
+  const handlePasswordChange = useCallback((e) => {
+    setPassword(e.target.value);
+  }, []);
+  const handleFormSubmit = async (e) => {
+    e.preventDefault();
+    dispatch({ type: LOG_IN_REQUEST });
+    try {
+      const response = await axios.post(
+        'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/signin',
+        {
+          email: email,
+          password: password,
+        }
+      );
```

Comment:
```suggestion
   const url = 'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/signin';
   const data = { email: email, password: password};
   try {
      const response = await axios.post( url, data );
```
이렇게 url이랑 data를 따로 분리해주면 어떨까요?